# o try é uma exerção para quando o bloco de codigos ocorrer um erro ele executar o `except` que é a exeção
try:
    print('Olá ')
except:
    print('A variavel x não está definida')
finally:
    print('f')